bundle/edu.gemini.obslog/src/main/java/edu/gemini/obslog/instruments/Igrins2LogSegment.java

```diff
@@ -17,7 +17,7 @@
 public class Igrins2LogSegment extends InstrumentLogSegment {
     public static final Logger LOG = Logger.getLogger(Igrins2LogSegment.class.getName());
 
-    private static final String NARROW_TYPE = "Igrins2";
+    private static final String NARROW_TYPE = "IGRINS-2";
```

Comment:
Does this have to match the `SpComponentType`?  If so, maybe we should just take it from there:
```java
private static final String NARROW_TYPE = SpComponentType.INSTRUMENT_IGRINS2.narrowType;
```

Reply:
Right, let me try that